source/_integrations/philips_js.markdown

```diff
@@ -133,3 +133,49 @@ Limits:
  - The integration does not expose current ambilight measured values since it would
 overload the event bus in Home Assistant.
  - There is no support to control the standard, non-expert, styles of the TV.
+
+## Selecting apps, channels, or favorites
+
+Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your TV. To capture the `content_id` for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
```

Comment:
I can't accept adding something like this to docs. You should be able to get the content id from the attributes on the state in developer console.

Reply:
What I mean is, accept anything that require turning on debug logs for the device.

Reply:
Unfortunately you cannot get the content_id from the state in the developer console. That would be nice, but it doesn't work that way. It shows me this:

```
activity_list: Amazon Prime, Plex, PBS, HBO, Xbox, TV1 Russia, Hulu, Netflix, PBS Kids, External Media, PlayStation, Czech TV, Curiosity Stream, Nintendo, YouTube, Google Play Movies, Home Screen, Apple TV
current_activity: HBO
activity_starting: null
devices_list: DVD, Old Mac Mini, Sky HD, AwakeLion AV Switch, Philips TV Apps 2, Xbox-SystemOS, XMBC, Plex Player, Chromecast, Russian Satellite, Philips TV, Philips TV apps, Game Console, Mac Mini (2010)
last_activity: HBO
friendly_name: harmony
supported_features: 4
```

None of those can be used to launch the activity.

As I am not the developer for this component, there's not much I can do to fix that, other than just take out the information about how I got the correct content_id for launching the apps. It would be a disservice to anyone who wants to use this to launch an app other than the ones I listed, but I suppose something is better than nothing. Is that what I should do?